src/content/resources/faq.md

```diff
@@ -799,6 +817,11 @@ Flutter and Dart don't offer application binary interface (ABI)
 compatibility. Offering ABI compatibility is not a current
 goal for Flutter or Dart.
 
+### How does Flutter handle scrolling?
+
+A custom scrolling implementation is used for each app platform so that
+scrolling matches that platform's native scrolling look and feel.
```

Comment:
There used to be a page that described that in depth. I can't find it, but maybe @parlough can. Xiao wrote it originally. Do you remember, Parker? If so, it might be good to link to that here.

Reply:
This one? https://docs.flutter.dev/ui/layout/scrolling